tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -619,209 +567,214 @@ describe(__filename, () => {
       fetchUserReviews({
-        errorHandlerId: errorHandler.id,
+        errorHandlerId: createErrorHandlerId({ userId }),
         page: newPage,
         userId: user.id,
       }),
     );
   });
 
   it('fetches reviews if user is loaded', () => {
-    const { params, store } = signInUserWithUserId(100);
+    const userId = 100;
+    const { params } = signInUserWithProps({ userId });
     const user = getCurrentUser(store.getState().users);
 
-    const dispatchSpy = sinon.spy(store, 'dispatch');
-    const errorHandler = createStubErrorHandler();
+    const dispatch = jest.spyOn(store, 'dispatch');
 
     const page = 123;
-    const location = createFakeLocation({ query: { page } });
+    const location = getLocation({ userId, search: `?page=${page}` });
 
-    renderUserProfile({ errorHandler, location, params, store });
+    renderUserProfile({ location, params });
 
-    sinon.assert.calledWith(
-      dispatchSpy,
+    expect(dispatch).toHaveBeenCalledWith(
       fetchUserReviews({
-        errorHandlerId: errorHandler.id,
-        page,
+        errorHandlerId: createErrorHandlerId({ userId }),
+        page: `${page}`,
         userId: user.id,
       }),
     );
   });
 
   it('does not fetch reviews if already loaded', () => {
-    const { params, store } = signInUserWithUserId(100);
+    const { params } = signInUserWithProps({ userId: 100 });
     const user = getCurrentUser(store.getState().users);
 
     _setUserReviews({ store, userId: user.id });
 
-    const dispatchSpy = sinon.spy(store, 'dispatch');
+    const dispatch = jest.spyOn(store, 'dispatch');
 
-    renderUserProfile({ params, store });
+    renderUserProfile({ params });
 
-    sinon.assert.neverCalledWithMatch(dispatchSpy, {
-      type: FETCH_REVIEWS,
-    });
+    expect(dispatch).not.toHaveBeenCalledWith(
+      expect.objectContaining({ 'type': FETCH_USER_REVIEWS }),
+    );
+    expect(dispatch).not.toHaveBeenCalledWith(
+      expect.objectContaining({ 'type': FETCH_REVIEWS }),
+    );
   });
 
   it(`displays the user's reviews`, () => {
     const userId = 100;
-    const { params, store } = signInUserWithUserId(userId);
+    const { params } = signInUserWithProps({ userId });
 
     const review = fakeReview;
     const reviews = [review];
     _setUserReviews({ store, userId, reviews });
-    const location = createFakeLocation({ query: { foo: 'bar' } });
 
-    const root = renderUserProfile({ location, params, store });
+    renderUserProfile({ params });
 
-    expect(root.find('.UserProfile-reviews')).toHaveLength(1);
-    expect(root.find('.UserProfile-reviews')).toHaveProp(
-      'header',
-      'My reviews',
-    );
-    expect(root.find('.UserProfile-reviews')).toHaveProp('footer', null);
-
-    expect(root.find(AddonReviewCard)).toHaveProp(
-      'review',
-      createInternalReview(review),
-    );
+    expect(screen.getByText('My reviews')).toBeInTheDocument();
+    expect(screen.getByText('It is Okay')).toBeInTheDocument();
```

Comment:
I assume this text ("It is Okay") comes from `fakeReview`? If so, I think it would be better to use `fakeReview.body` instead.